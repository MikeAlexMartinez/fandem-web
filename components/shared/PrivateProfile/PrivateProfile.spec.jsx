import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PrivateProfileComponent from './PrivateProfile';

describe('<PrivateProfile />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<PrivateProfileComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
