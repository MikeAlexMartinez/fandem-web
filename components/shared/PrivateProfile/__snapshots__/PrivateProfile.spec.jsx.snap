// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivateProfile /> renders and matches the snapshot 1`] = `
<div>
  Private Profile
</div>
`;
